src/frontend/komponenter/Fagsak/FagsakContainer.tsx

```diff
@@ -11,13 +11,14 @@ import Dokumentutsending from './Dokumentutsending/Dokumentutsending';
 import JournalpostListe from './journalposter/JournalpostListe';
 import Personlinje from './Personlinje/Personlinje';
 import Saksoversikt from './Saksoversikt/Saksoversikt';
+import { BehandlingProvider } from '../../context/behandlingContext/BehandlingContext';
 import { DokumentutsendingProvider } from '../../context/DokumentutsendingContext';
 import { useFagsakContext } from '../../context/fagsak/FagsakContext';
 import useSakOgBehandlingParams from '../../hooks/useSakOgBehandlingParams';
 import { useAmplitude } from '../../utils/amplitude';
 
 const HovedInnhold = styled.div`
-    height: calc(100vh - 6rem);
+    height: calc(100vh - 3rem);
```

Comment:
Nå er Personlinjal, som er 3rem høy, inne i HovedInnhold. Trenger derfor ikke å trekke fra 3 rem for den.